moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -42,72 +38,25 @@ export const signalToFeatureCollection = signal => {
     / geometry will be saved.
     */
     const featureUUID = signal?.id || uuidv4();
-    const feature = {
-      type: "Feature",
-      properties: {
-        ...signal?.properties,
-        renderType: "Point",
-        // I was considering changing this to be INTERSECTIONID, but since the sourceLayer is being saved
-        // as "drawnByUser" and not "ATD_ADMIN.CTN_Intersections", I've left it as PROJECT_EXTENT_ID
-        PROJECT_EXTENT_ID: featureUUID,
-        sourceLayer: "drawnByUser",
+    const featureSignalsRecord = {
+      // MultiPoint coordinates are an array of arrays, so we wrap the coordinates
+      geography: {
+        ...signal.geometry,
+        type: "MultiPoint",
+        coordinates: [signal.geometry.coordinates],
       },
-      geometry: signal.geometry,
-      id: featureUUID,
+      knack_id: signal.properties.id,
+      location_name: signal.properties.location_name,
+      signal_type: signal.properties.signal_type,
+      signal_id: signal.properties.signal_id,
+      // I was considering changing this to be INTERSECTIONID, but since the sourceLayer is being saved
+      // as "drawnByUser" and not "ATD_ADMIN.CTN_Intersections", I've left it as PROJECT_EXTENT_ID
+      project_extent_id: featureUUID,
+      source_layer: "drawnByUser",
     };
-    featureCollection.features.push(feature);
-  }
-  return featureCollection;
-};
-
-export const useInitialSignalComponentValue = (
```

Comment:
React Hook Form takes over the state management for the signal autocomplete now so this helper is no longer needed.